backend/prompt_studio/prompt_studio_core/prompt_studio_helper.py

```diff
@@ -259,6 +259,13 @@ def get_select_fields() -> dict[str, Any]:
         choices = f.read()
         f.close()
         response: dict[str, Any] = json.loads(choices)
+
+        for modifier_plugin in modifier_loader:
+            cls = modifier_plugin[ModifierConfig.METADATA][
+                ModifierConfig.METADATA_SERVICE_CLASS
+            ]
+            response = cls.update_select_choices(default_choices=response)
```

Comment:
@harini-venkataraman this might work because we have `update_select_choices()` defined in the plugin.However if we add any other modifier plugin which doesn't have that function - it'll break.

I hope these modifier plugins are derived from a generic interface that exposes such methods that needs to be implemented. If not, please mark it as a TODO item where `modifier_loader` gets assigned.
Also it'll be good to rename the variable to `modifier_plugins` as discussed before - please mark that as a TODO item as well similarly

Reply:
@chandrasekharan-zipstack Thanks for pointing it out. Have handled the rename of variables. 
Wrt having a base class for plugins, it is yet to be picked, marked with TODO.